banners/templatetags/banners.py

```diff
@@ -0,0 +1,23 @@
+from django import template
+from django.template.loader import render_to_string
+
+from banners.models import Banner
+
+register = template.Library()
+
+
+@register.simple_tag
+def render_active_banner(psf_pages_only=True):
```

Comment:
What about having two template tags `render_active_banner` and `render_active_psf_banner` instead of adding a boolean argument? This would also make it clearer what is going on in the template.

Reply:
great suggestion, thank you!